### PR TITLE
Adding Success and Pending Minting Dialogs

### PR DESCRIPTION
**Pull request by iamadilmughal**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,39 +1,156 @@
...
+  value: string;
+  isProminant?: boolean;
+};
 
 function App() {
   const [selectedDate, setSelectedDate] = useState<number>();
-  const [isOpenDialog, setIsOpenDialog] = useState(false);
+  const [openDialogType, setOpenDialogType] = useState<string | null>(null);
+
+  const dialogVals: MintingProcessInfo[] = [
```
Comment (meho99) [inline src/App.tsx:30]: Both success and progress dialogs shows the same information. What do you think about creating a common component that will contain info about the deposit and than use it in the content for both of them? 
image

File: src/App.tsx [hunk 2]
```diff
@@ -1,39 +1,84 @@
...
 import { DashboardContainer } from './pages/Dashboard/Dashboard.container';
 import { ProposalDetailsContainer } from './pages/ProposalDetails/ProposalDetails.container';
 import { ProposalsListContainer } from './pages/ProposalsList/ProposalsList.container';
 import { StakingContainer } from './pages/Staking/Staking.container';
-import { DateSelector } from './components/DateSelector/DateSelector.component';
 
 function App() {
   const [selectedDate, setSelectedDate] = useState<number>();
-  const [isOpenDialog, setIsOpenDialog] = useState(false);
+  const [openDialogType, setOpenDialogType] = useState<string | null>(null);
```
Comment (michalsidzej) [inline src/App.tsx:23]: let's use union as a type here. Also (that's completely personal opinion, and discussion purpose) I don't really like using `null` and `undefined` in the same apa. As React is evaluating empty state to `undefined`, I would argue for not using `null` at all.
```suggestion
  const [openDialogType, setOpenDialogType] = useState<'success' | 'error'>();
```

File: src/components/MintingInfo/MintingInfo.component.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          my: '5px',
+        }}
+      >
+        <Typography
+          variant="h4"
+          sx={{
+            textAlign: 'right',
+          }}
+        >
+          {d.label} :
```
Comment (michalsidzej) [inline src/components/MintingInfo/MintingInfo.component.tsx:24]: I think we should't have this space here
```suggestion
          {d.label}:
```

File: src/components/MintingInfo/MintingInfo.component.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import { MintingInfoPros } from './MintingInfo.types';
+
+export const MintingInfo = ({ data }: MintingInfoPros) => (
+  <>
+    {data.map((d, i) => (
```
Comment (michalsidzej) [inline src/components/MintingInfo/MintingInfo.component.tsx:8]: What do you think about that? Just a suggestion
```suggestion
    {data.map(({ label, value, isProminant }, index) => (
```

**Review by michalsidzej — APPROVED** [review]

**Pull request merged**